### ISSUE TITLE
Feature request: Conda package

### ISSUE THREAD

**Issue by villares** [p5py/p5#411]
I see many projects that have "non-python" dependencies, like is the case of GLFW for p5py, to use Anaconda/miniconda as an install option, as it is able to install more stuff in your system, in a single step.

Have you considered having a conda package?

**Comment by tushar5526** [comment]
Hey, nice idea. I think this will make the installation process easier, given we have external skia dependencies as well. Thoughts @ziyaointl

**Comment by ziyaointl** [comment]
Yeah I think that's a good idea! I'm not very familiar with conda myself, but I'm happy to try out the installation on multiple platforms if someone opens a PR for this.

**Comment by SarthakNikhal** [comment]
I can try creating a conda package. can you assign this issue to me and explain what should the conda package do

**Comment by tushar5526** [comment]
Hey, I have not played around with conda packages much but I believe it should solve the problem of manually installing external libraries viz `glfw`. Accepted solution would be to have a package that install p5py and all the external "non-python" dependencies. You can refer the comments above as well.
